src/centengine/engine_impl.cc

```diff
@@ -41,10 +41,14 @@
 #include "com/centreon/engine/hostgroup.hh"
 #include "com/centreon/engine/logging.hh"
 #include "com/centreon/engine/logging/logger.hh"
+#include "com/centreon/engine/modules/external_commands/internal.hh"
+#include "com/centreon/engine/modules/external_commands/processing.hh"
+#include "com/centreon/engine/modules/external_commands/utils.hh"
 #include "com/centreon/engine/service.hh"
```

Comment:
We have an issue here, we cannot use functions declared in external_commands. We have to keep a copy of the functions used from external_command in command_manager or in another place.
One day, external_command will disappear.